virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -85,6 +85,8 @@ The SVVP Certification applies to:
 
 //CNV-28729 Release note: New
 * You can use a xref:../../virt/virtual_machines/advanced_vm_management/virt-schedule-vms.adoc#virt-vm-custom-scheduler_virt-schedule-vms[custom scheduler] to schedule a virtual machine (VM) on a node.
+//CNV-29048 Release note: NEW Metrics name changes
+* Many OpenShift Virtualization metrics have changed or will change in a future version. These changes could affect your custom dashboards. See link:https://access.redhat.com/articles/7028805[OpenShift Virtualization 4.14 metric changes] for details.
```

Comment:
@aspauldi the linked document has "PRIVATE: This document is for internal use and is not accessible to customers." header, is it appropriate for release note, that would be out for public consumption?

@sradco https://access.redhat.com/articles/7028805 includes metrics that would be renamed in 4.15. Referencing it without clearly mentioning which are getting updated in 4.14 vs 4.15, would be more confusing I feel.

Reply:
@dbasunag The linked article has not yet been published.

@sradco Is it ready for publication?

Reply:
@apinnick I see the article https://access.redhat.com/articles/7028805 is published, but some of the metrics that are mentioned there did not get renamed till 4.15. This would be misleading. @sradco what do you suggest we do?